### ISSUE TITLE
[Bug]: twig sandbox causes error when sending emails after update

### ISSUE THREAD

**Issue by passioneight** [pimcore/pimcore#13788]
### Expected behavior

When updating from `10.5.8` to >`10.5.9` (in my case `10.5.12`), emails should be sent just as before.

### Actual behavior

After the update, emails were not sent. Instead the error in the following line was thrown:

https://github.com/pimcore/pimcore/blob/f2fa96ee1db3e0ef5ca4cd8a47c4d8aabe5d9d1b/lib/Mail.php#L667

Which stated that the `default` filter is not allowed. 

Note that adding the following config fixed the issue for me:
```
 pimcore:
    templating_engine:
        twig:
            sandbox_security_policy:
                filters: ['default']
```

### Steps to reproduce

1. Create E-Mail Template
2. Add WYSIWYG
3. Add content like the one in the screenshot
4. Send Test-Email

<img width="326" alt="image" src="https://user-images.githubusercontent.com/57256630/207337834-fe4dd965-6f51-49e0-9255-2159277de0ca.png">

Reproducible on demo:

![image](https://user-images.githubusercontent.com/57256630/207355950-bae5bfb4-748a-456f-9d72-e639ea842c52.png)

**Comment by dvesh3** [comment]
@passioneight Starting with `10.5.9`, you need to whitelist twig tags and filters in the config, if you want to use them on mail templates mainly used for params rendering. please see [upgrade notes](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html#page_10-5-9).

**Comment by passioneight** [comment]
@dvesh3 I'm not sure if we should allow breaking changes in minor versions, just because they are mentioned in the upgrade notes.

**Comment by dvesh3** [comment]
It is a bug fix https://github.com/pimcore/pimcore/security/advisories/GHSA-5qxq-vgmm-q39m

**Comment by passioneight** [comment]
@dvesh3 yes and an important one too. Yet, updating minor versions should not break functionality of existing projects. One could have added the functionality with all filters allowed first (for the minor version) and restricted the filters with `v11`, for example.

**Comment by dvesh3** [comment]
I understand your concern completely and we had the same thought initially to make it a smooth upgrade. 
However, we couldn't come up with a better solution than using Twig Sandbox extension, which restricts everything by default to render untrusted code and it can't be setup the other way to allow all filters and then restrict some. 

And we also concluded, it is easy for implementations to whitelist some that are used in the templates than core whitelisting all (might resurface the issue again).

If you have a better idea to fix the vulnerability, we are more than happy to change it.

**Comment by passioneight** [comment]
Well, obviously, I'd have gone down the path I suggested before as it first prepares the issue to be fixed and then fixes it with a BC in a major version. But I can see why you would want it fixed right away.

Maybe we can instead update the upgrade notes with a highlighted note that this is a potential BC (depending on the e-mail template) and that it was consciously added. I think this would increase awareness.

Wdyt?

**Comment by fashxp** [comment]
>  One could have added the functionality with all filters allowed first 

That would not have fixed the security issue though! We know, it is a compromise, sometimes you have to make choices. 

Anyway, to be clear, it is not about the email template as a whole, it is only about the twig parameter feature for user entered content editables or subject ... which was never meant to use the full power of twig anyway. 

Let's add the `default` filter to the default whitelist. @passioneight can you provide a PR?

**Comment by dvesh3** [comment]
Fixed by #13794
